common/core/src/commonMain/kotlin/com/trafi/core/Logger.kt

```diff
@@ -1,5 +1,9 @@
 package com.trafi.core
 
+import io.ktor.client.features.logging.*
 import io.ktor.client.features.logging.Logger
 
-interface Logger : Logger
+interface Logger : Logger {
+    val apiLogLevel: LogLevel
```

Comment:
Let's create our own enum and map it to ktor's so we don't expose `Ktor_client_loggingLogLevel` to the Swift world 😄

Reply:
Something like this? 🤔 
```
package com.trafi.core

import io.ktor.client.features.logging.*
import io.ktor.client.features.logging.Logger
import io.ktor.client.features.logging.LogLevel as KtorLogLevel

interface Logger : Logger {

    val logLevel: LogLevel

    val apiLogLevel: KtorLogLevel
        get() = when (logLevel) {
            LogLevel.ALL -> KtorLogLevel.ALL
            LogLevel.INFO -> KtorLogLevel.INFO
            LogLevel.HEADERS -> KtorLogLevel.HEADERS
            LogLevel.BODY -> KtorLogLevel.BODY
            LogLevel.NONE -> KtorLogLevel.NONE
        }
}

enum class LogLevel {
    ALL,
    BODY,
    HEADERS,
    INFO,
    NONE;
}
```

Reply:
I had this implemented yesterday, but only pushed today.. 
but i like your mapping to KtorLogLevel better 👍